### DISCUSSION TITLE
Zenodo citations

### DISCUSSION THREAD

**Discussion by dfgordon** [tree-sitter/tree-sitter#1000] [Ideas] [answered]
Hello, have you considered making this software citable, e.g., via CERN's zenodo system [https://zenodo.org/] ?
I tried searching there but didn't find it.
I want to cite the software in a journal article.
Thanks!

**Comment by maxbrunsfeld** [comment, marked as answer]
Thanks for the suggestion; I had not heard of Zenodo before. I added a Zenodo badge to the README.

**Reply by dfgordon** [reply]
That takes care of it, thanks for the quick action.
